how-to/onboarding-guide.md

```diff
@@ -147,9 +148,13 @@ the `title:` yaml element and update `contributor_type:` as see below:
 
 * If they haven't already done, ask the new editor turn on [two-factor authentication (2FA) for GitHub](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa).
 
+* Give editors permissions they will need on GitHub to manage reviews:
+  * For ad-hoc guest editors: invite the editor to the `software-submission` repository
+    with "maintain" permissions.
+  * For new editors (not ad-hoc): invite editors to the pyOpenSci GitHub organzation
```

Comment:
ok you have the specifics down here - i should have read through the entire thing... we may be able to combine some of this but i just think because people skim it's good to be specific where we can.